samples/cellular/azure_fota/prj.conf

```diff
@@ -67,8 +67,7 @@ CONFIG_NET_NATIVE=n
 CONFIG_NRF_MODEM_LIB=y
 
 # newlibc
```

Comment:
This comment should be removed or updated.

Reply:
Thanks, I have updated that